guides/Advanced SettingGateway/UnderstandingSchemaPieces.md

```diff
@@ -0,0 +1,67 @@
+# Understanding Schema's Keys
+
+As mentioned in the previous tutorial, {@tutorial UnderstandingSchema}, SettingGateway's schema is divided in two parts: **folders** and **pieces**. Pieces are contained into folders, but it cannot have keys nor folders, instead, this holds the key's metadata such as its type, if it's configurable by the configuration command... you can check more information in the documentation: {@link SchemaPiece}.
```

Comment:
As I said in Discord, calling items in the schema as `**folders** and **pieces**` and also calling `commands, events, extendables, inhibitors`  'pieces' as well might be confusing to new users

Pieces are contained ***in*** folders, but ***they*** cannot have keys nor folders. [PERIOD] ***Instead*** .....